nautobot/core/templates/inc/nav_menu.html

```diff
@@ -837,6 +847,37 @@
                     </span>
                 </div>
             </form>
+            {% comment %}
+            The following if statement means the Query tab is only rendered when the GraphiQL
+            is rendered.
+            {% endcomment %}
+            {% if  saved_graphiql_queries and perms.extras.view_graphqlquery %}
+
+                <ul class="nav navbar-nav navbar-right">
+                    <li class="dropdown">
+                        <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
+                            <span>Queries </span>
+                            <span class="caret"></span>
+                        </a>
+                        <ul class="dropdown-menu" id="graphql-queries-menu">
+                            <li>
+                                <a href="#" onclick="reload('', '')">None</a>
```

Comment:
This has never seemed quite intuitive to me in the demos you've given. Is it even needed at all? Or should it be changed perhaps to a "Clear" button instead of a "None" menu item?

Reply:
sorry, i put in the change to call it clear. I think it useful as some queries could be quite long so having the clear button lets you empty both the query box and the variable box.

Reply:
OK, but should it be a button instead of a list item link?

Reply:
yeah thats a good idea